lib/src/Dscp/Resource/Keys/Functions.hs

```diff
@@ -0,0 +1,127 @@
+-- | Functions to work with key storage.
+-- Later this file will also provide possibility to choose a key from
+-- the predefined test genesis instead of providing a key explicitly.
+
+module Dscp.Resource.Keys.Functions
+    ( linkStore
+    ) where
+
+import Data.Aeson (eitherDecode', encode)
+import qualified Data.ByteString.Lazy as LBS
+import Fmt ((+||), (|+), (||+))
+import Loot.Log (MonadLogging, logDebug, logInfo)
+import System.Directory (doesFileExist)
+import System.Directory as D
```

Comment:
First line 
```haskell
import System.Directory (doesFileExist)
```
does nothing, as second one imports `doesFileExist` in both qualified and unqualified modes.

Reply:
Pff, right :man_facepalming: